include/ginkgo/core/matrix/dense.hpp

```diff
@@ -128,8 +128,12 @@ class Dense : public EnableLinOp<Dense<ValueType>>,
      */
     static std::unique_ptr<Dense> create_with_config_of(const Dense *other)
     {
-        return Dense::create(other->get_executor(), other->get_size(),
-                             other->get_stride());
+        // De-referencing `other` before calling the functions (instead of
```

Comment:
We should add a TODO or some kind of mark here, so that we remember to fix it in the future and should also probably mention it somewhere in the known issues during the release.

Reply:
For users, this should not matter as it does exactly the same as it did previously. However, other developer probably have a very hard time figuring out why parts of their code in `.cu` files is not running when they use any `static` functions.
Do we have a place where we put the known issues? I think mentioning it there would be useful because I don't think CUDA will fix that before our release.

I will also add a `TODO` to check if it is still a problem.

Reply:
I had created a wiki page though it is now empty.
https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues

Reply:
I just added this TODO:
```C++
// TODO Check if the compiler error is fixed and revert to `operator->`.
```
Do you think this is fine?

Reply:
I also just added a description to the [known issues wiki page](https://github.com/ginkgo-project/ginkgo/wiki/Known-Issues).

Reply:
Looks good!